queries/heex/injections.scm

```diff
@@ -1,10 +1,17 @@
-; HEEx directives can span multiple interpolated lines of Elixir
-(directive [
-  (expression_value) 
-  (partial_expression_value)
-] @elixir @combined)  
+; directives are standalone tags like '<%= @x %>'
+;
+; partial_expression_values are elixir code that is part of an expression that
+; spans multiple directive nodes, so they must be combined. For example:
+;     <%= if true do %>
+;       <p>hello, tree-sitter!</p>
+;     <% end %>
+(directive (partial_expression_value) @elixir @combined)  
 
-; HEEx Elixir expressions are always within a tag or component
+; Regular expression_values do not need to be combined
+(directive (expression_value) @elixir)
+
+; expressions live within HTML tags, and do not need to be combined
+;     <link href={ Routes.static_path(..) } />
```

Comment:
For easy reviewing, a reference of the upstream version with updated syntax:

https://github.com/phoenixframework/tree-sitter-heex/blob/main/queries/injections.scm

<img width="661" alt="image" src="https://github.com/nvim-treesitter/nvim-treesitter/assets/1214337/78f897f2-9b61-4ea4-86d1-b8c7abb5e79a">

Reply:
Could you please just paste this as a codeblock? Screenshots are hard to work with (e.g., for later copy&pasting).

Reply:
done

Reply:
And now _we_ switched to the upstream format, so you can rebase on `master` and use the upstream version :)

Reply:
Done :)